lib/communityFields.js

```diff
@@ -0,0 +1,31 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+
+import { fetchCommunity, fetchEventById } from './fetch';
+import { EventType } from './eventFields';
+
+const communityType = new GraphQLObjectType({
+  name: 'Community',
+  fields: {
+    communitySummary: { type: GraphQLString },
```

Comment:
@lpil communitySummary, my bad for calling it summary yesterday. It will be a bit tricky to change this in prismic